actions/distribute_workers.py

```diff
@@ -27,14 +29,12 @@ def __init__(self, ai):
 
     async def handle(self, iteration):
         """Groups the resulting actions from all functions below"""
+        self.drone_dodge()
```

Comment:
not sure if we will ever dodge, because it will be overwritten by the distribution actions

Reply:
Ill remove it until we find a way to do it correctly